USER_MANUAL.md

```diff
@@ -212,5 +213,16 @@ Actors used in gyms are in `Contetnt\Actors`: add any new Actors to this directo
 * Adjust the setting "SpatialOS Settings -> Debug -> Spatial Debugger Class Path" to `BP_VerboseSpatialDebugger`.
 * If it is working correctly, you will see "10 10 10" and "20 20 20" appear over the top of each cube intermitantly. This represents the HitLocation data being sent using a cross server RPC inside a PointDamageEvent object and the Origin of RadialPointDamage event. 
 
+##### Teleporting gym.
+* Tests actor migration when load balancing is enabled.
+* NOTE : This gym is likely to have random failures, as we are still working on load balancing.
```

Comment:
Can you list any known issues? I feel that if I was testing this otherwise I'd just have to dm you asking